doc/rados/operations/monitoring.rst

```diff
@@ -276,22 +276,36 @@ three monitors may return the following:
 	        0,
 	        1,
 	        2],
+	  "quorum_names":[
+		"a",
+		"b",
+		"c"],
+	  "quorum_leader_name":"a",
```

Comment:
Maybe I'm wrong, but I think **after** each colon there will be a space. This would apply to this line, but also all the other lines where you added a json output with colons.

Reply:
Yes, you are right @jecluis. That practice is being followed in the doc, which is different from the actual json output. Fixed it now. Thanks.